justfile

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env just --justfile
```

Comment:
Justfiles look appealing, but would you mind if I'll convert this to a Makefile after merge instead? It's just because Makefiles are ubiquitous.

Reply:
Sure, fine by me. For something simple like this, it's probably not hard, but I've been bitten by make files depending on gnu extensions so often (Mac comes with an old, non gnu make impl) that I figured I'd give just a try.